csharp/sources/Valkey.Glide/Commands/IStringBaseCommands.cs

```diff
@@ -40,4 +40,123 @@ public interface IStringBaseCommands
     /// Otherwise, returns <see langword="null" />.
     /// </returns>
     Task<GlideString?> Get(GlideString key);
+
+    /// <summary>
+    /// Retrieves the values of multiple <paramref name="keys" />.
+    /// </summary>
+    /// <remarks>
+    /// <example>
+    /// <code>
+    /// GlideString[] keys = { "key1", "key2", "key3" };
+    /// GlideString?[] values = await client.MGet(keys);
+    /// Console.WriteLine(values[0]); // Output: value1 or null if key1 doesn't exist
+    /// </code>
+    /// </example>
+    /// </remarks>
+    /// <param name="keys">A list of keys to retrieve values for.</param>
+    /// <returns>
+    /// An array of values corresponding to the provided <paramref name="keys" />.<br/>
+    /// If a key is not found, its corresponding value in the list will be <see langword="null" />.
+    /// </returns>
+    /// <seealso href="https://valkey.io/commands/mget/">valkey.io</seealso>
+    Task<GlideString?[]> MGet(GlideString[] keys);
```

Comment:
Please follow SER naming
args, arg types, return type and function name is differernt.
Applicable to all